### PR TITLE
Curl

### PR DESCRIPTION
**Pull request by brson**
Supercedes https://github.com/rust-lang-nursery/rustup.rs/pull/434

### CODE REVIEW

**Comment by brson** [comment]
@alexcrichton [Fresh build errors](https://ci.appveyor.com/project/brson/rustup-rs/build/1.0.202/job/hvhsx30rsa9s4qki).

**Comment by brson** [comment]
Seems like it can't find 'make'?

**Comment by alexcrichton** [comment]
Yeah I think that's the fix, although I believe the path is `C:\msys64\usr\bin` instead of `C:\msys64\mingw$BITS\usr\bin`

**Comment by brson** [comment]
Failing for yet new reasons.

File: appveyor.yml [hunk 1]
```diff
@@ -21,6 +24,12 @@ on_finish:
   - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 
 install:
+  - if defined MSYS_BITS set PATH=C:\msys64\mingw%MSYS_BITS%\bin;C:\msys64\usr\bin;%PATH%
```
Comment (alexcrichton) [inline appveyor.yml]: s/MSYS_BITS/BITS

**Pull request closed without merging**